metasv/generate_sv_intervals.py

```diff
@@ -201,6 +201,18 @@ def find_other_bp(aln, isize_mean, isize_sd, svtype, soft_clip_location, num_sd=
             #if other_bp == -1:
             #    print aln, dist_L_end, dist_R_end, aln.is_reverse, aln.pos > aln.pnext, other_bp
             return other_bp
+    elif svtype == "DUP;ITX":
+        soft_clip_tuple = find_softclip(aln)
+        if soft_clip_tuple is not None:
```

Comment:
None evaluates to false so the check can just be `if soft_clip_tuple`. Similarly, for the other svtype branches.